docs/modules/user-guide/partials/proc_migrating-plug-ins.adoc

```diff
@@ -2,20 +2,20 @@
 = Migrating plug-ins
 
 [role="_abstract"]
-This section describes how to migrate plug-ins to devfile v2.x. Plug-ins are now specified using a devfile. A plug-in specified in a v1.0 `meta.yaml` does not work as it is in a devfile v2.x.
+This section describes how to migrate plug-ins to devfile v2.x. A plug-in specified in a meta.yaml in a devfile v1.x no longer works in a devfile v2.x.
```

Comment:
Looks like `meta.yaml` needs to be formatted.

Reply:
Done!